src/sentry/static/sentry/app/components/lastCommit.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import Avatar from './avatar';
+import TimeSince from './timeSince';
+
+import {t} from '../locale';
+
+const LastCommit = React.createClass({
+  propTypes: {
+    commit: React.PropTypes.object.isRequired,
+    headerClass: React.PropTypes.string
+  },
+
+  renderMessage(message) {
+    if (!message) {
+      return t('No message provided');
+    }
+
+    let firstLine = message.split(/\n/)[0];
+    if (firstLine.length > 100) {
+      let truncated = firstLine.substr(0, 90);
+      let words = truncated.split(/ /);
+      // try to not have elipsis mid-word
+      if (words.length > 1) {
```

Comment:
This has a couple of potential holes, but we can punt on them:

e.g. what if message is:

```
" a-really-long-message-preceded-by-a-space-pretend-is-90-chars"
```

It will become `" ..."`